vizro-core/examples/_dev/app.py

```diff
@@ -1,64 +1,49 @@
 """Dev app to try things out."""
 
-from typing import Optional
-
-import dash_bootstrap_components as dbc
 import pandas as pd
 import vizro.models as vm
 import vizro.plotly.express as px
-from dash import html
 from vizro import Vizro
-from vizro.figures import kpi_card
 from vizro.models.types import capture
 
-tips = px.data.tips
+df_stocks = px.data.stocks(datetimes=True)
+df_stocks_long = pd.melt(
+    df_stocks,
+    id_vars="date",
+    value_vars=["GOOG", "AAPL", "AMZN", "FB", "NFLX", "MSFT"],
+    var_name="stocks",
+    value_name="value",
+)
 
 
-@capture("figure")  # (1)!
-def custom_kpi_card(  # noqa: PLR0913
-    data_frame: pd.DataFrame,
-    value_column: str,
-    *,
-    value_format: str = "{value}",
-    agg_func: str = "sum",
-    title: Optional[str] = None,
-    icon: Optional[str] = None,
-) -> dbc.Card:  # (2)!
-    """Creates a custom KPI card."""
-    title = title or f"{agg_func} {value_column}".title()
-    value = data_frame[value_column].agg(agg_func)
+@capture("graph")
+def vizro_plot(data_frame, stocks_selected, **kwargs):
```

Comment:
what would be the best way to inform users that in this case, the `stocks_selected` should be a list? Because from user perspective, when they do 
```
options=[{"label": s, "value": s} for s in df_stocks_long["stocks"].unique()]
```
It's not intuitive to think `stocks_selected` should be a list right?

Reply:
I don't think we need to inform them - given that we stick with Dash default return type now, there should be no confusion hopefully. Other than that, there is always our logging message when you run the app in DEBUG mode, where you can also see the inputs/outputs of the actions. And if that doesn't help one can always check the input/outputs inside the custom chart function in DEBUG mode :)

<img width="734" alt="Screenshot 2024-07-03 at 18 28 12" src="https://github.com/mckinsey/vizro/assets/90609403/32f3b9b6-48c5-4f81-ba68-81ef7006f62f">

Reply:
I've changed the depth level of the logging messages now, so it's visible there:

<img width="1164" alt="Screenshot 2024-07-03 at 18 46 22" src="https://github.com/mckinsey/vizro/assets/90609403/d3a5e929-fc06-483b-b6f2-338f75a9e3c8">